plugin/pom.xml

```diff
@@ -76,6 +76,16 @@
 
     <dependency>
       <groupId>io.fabric8</groupId>
+      <artifactId>fabric8-maven-enricher-karaf</artifactId>
```

Comment:
In general I would create a separate module if that let me organize things better. For example, I was working on a spring-boot quickstart and noticed that yml properties are ignored when building the health checks. So I went to the "enrichers" directory expecting to find a spring-boot module... At the end I found where it was by searching the string "8081" and finding it in a comment :)

But you're right about the modules, it was almost impossible to build last Apache Camel after we have introduced other 200+ modules (also because of me :D). So also including them into standard or create a "stack" / "technology" module where things are separated into packages (not by type of enricher, but by technology stack) would be a good option. What do you think?